core/src/main/java/com/graphhopper/routing/util/spatialrules/countries/DefaultSpatialRule.java

```diff
@@ -0,0 +1,78 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for
+ *  additional information regarding copyright ownership.
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License. You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.graphhopper.routing.util.spatialrules.countries;
+
+import com.graphhopper.reader.ReaderWay;
+import com.graphhopper.routing.util.spatialrules.AbstractSpatialRule;
+import com.graphhopper.routing.util.spatialrules.AccessValue;
+
+/**
+ * Default Implementation for the SpatialRule that contains the current default Values
+ */
+public class DefaultSpatialRule extends AbstractSpatialRule {
```

Comment:
Not sure. The idea was to define default rules for streets that are found frequently in different countries. Adding a default CountryIsoA3Name might now work? I might make sense to make this class abstract as well. The idea was to provide defaults, that why I called it Default. Should we move it to the AbstractSpatialRule?